docs/feature-store/sources-targets.md

```diff
@@ -0,0 +1,47 @@
+| [mlrun.datastore.ParquetSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.ParquetSource)    | Reads the Parquet file/dir as the input source for a flow.                      | Y      | Y     | Y      |batch|
+| [mlrun.datastore.StreamSource](../api/mlrun.datastore.html#mlrun.datastore.StreamSource)                          | Sets the stream source for the flow. If the stream doesn’t exist it creates it. | Y      | N     | N      |event|
+
+## Targets
+| Class name                                                                                                    | Description                                                                                        | storey | spark | pandas |Online or offline|
+| --------------------------------------------------                                                            | -------------------------------------------------------                                            | ---    | ---   | ---    |---|
+| [mlrun.datastore.CSVTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.CSVTarget)        | Writes events to a CSV file.                                                                       | Y      | Y     | Y      |offline|
+| [mlrun.datastore.KafkaTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.KafkaTarget)    | Writes all incoming events into a Kafka stream.                                                    |  |  |  |  offline|
+| [mlrun.datastore.NoSqlTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.NoSqlTarget)    | Persists the data in V3IO table to its associated storage by key.                                  | Y      | Y     | Y      |Online (default)|
+| mlrun.datastore.RedisNoSqlTarget                                                                              | Persists the data in Redis table to its associated storage by key.                                 | Y      | Y     | N      |Online|
+| mlrun.datastore.SqlTarget                                                                                     | Persists the data in SQL table to its associated storage by key.                                   | Y      | N     | Y      |Offline (default)|
+| [mlrun.datastore.ParquetTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.ParquetTarget)| The Parquet target storage driver, used to materialize feature set/vector data into parquet files. | Y      | Y     | Y      |Online|
+| [mlrun.datastore.StreamTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.StreamTarget)  | Writes all incoming events into a V3IO stream.                                                     | Y      | N     | N      |Offline|
+
+## Partitioning on Parquet target
```

Comment:
we should mention the 1024 partitions here.

Reply:
@assaf758 - what are 1024 partitions?